blocks/ad-taboola-block/features/ad-taboola/default.jsx

```diff
@@ -79,7 +80,16 @@ class AdTaboola extends Component {
       return;
     }
 
-    this.appendScript('tbl-loader', head, () => taboolaLoader(this.publisherID));
+    const { metaValue } = this.props;
+
+    const pageType = metaValue('page-type');
```

Comment:
Do we need a fallback for this?  I assume not, but can metaValue return a falsey value?

If not, You can resolve this.  I see no other issues and the tests were fine.

Reply:
that's a good question. I can try to find docs for this I was working under the assumption that it'd always be a string. if it was an empty string (falsy), I'm not sure if that would hurt anything. the object TABOOLA_TO_PAGE_TYPE would return a falsy value ''